packages/paste-core/components/skeleton-loader/__tests__/index.spec.tsx

```diff
@@ -118,8 +118,10 @@ describe('SkeletonLoader', () => {
   describe('Accessibility', () => {
     it('Should have no accessibility violations', async () => {
       const {container} = render(<SkeletonLoader />);
+
       const results = await axe(container);
-      expect(results).toHaveNoViolations();
+
+      await waitFor(async () => expect(await results).toHaveNoViolations());
```

Comment:
thought: `await results` might not be needed, since results is already an awaited result on line 122. 

question: why do we need the rest of the wrapping async code?